pennylane/qchem/convert.py

```diff
@@ -330,6 +330,7 @@ def import_operator(qubit_observable, format="openfermion", wires=None, tol=1e01
     >>> print(h_pl)
     (-0.0548 * X(0 @ X(1) @ Y(2) @ Y(3))) + (0.14297 * Z(0 @ Z(1)))
     """
+    format = format.strip().lower()
```

Comment:
It might also be needed in `qml.qchem.qubit_observable` and `qml.qchem.hf_state.html`.
Moreover `qml.qchem.observable` can be cleaned not to use the stripping multiple times.